packages/docs-page/index.js

```diff
@@ -38,6 +39,7 @@ export function DocsPageWrapper({
   // TEMPORARY (https://app.asana.com/0/1100423001970639/1160656182754009)
   // activates the "jump to section" feature
   useEffect(() => {
+    if (!document) return
```

Comment:
great callout, left this in by accident - had been trying to debug something earlier 🤦 since realized this is pretty ill-informed as useEffect would only be called on the client-side (i think?) so check is unnecessary. Have cut back out 🔪 ✅